backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentDatabaseHandler.java

```diff
@@ -706,6 +701,13 @@ private ReleaseLink getReleaseLink(Release release) {
         }
         ReleaseLink releaseLink = new ReleaseLink(release.id, fullname, release.name, release.version);
         releaseLink.setClearingState(release.getClearingState());
+        List<Attachment> licenseInfoAttachments = nullToEmptySet(release.getAttachments()).stream()
```

Comment:
There has to be a better alternative to "calculate the `licenseInfoAttachments`" whenever someone asks `getReleaseLink(..)`